### PR TITLE
add apply flag: inventory-policy

### PR DESCRIPTION
**Pull request by Liujingfang1**

### CODE REVIEW

**Comment by Liujingfang1** [comment]
@mortent @seans3

**Comment by mortent** [comment]
Small comment, but mostly looks good.

File: pkg/apply/destroyer.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ func (d *Destroyer) Initialize() error {
 // Run performs the destroy step. Passes the inventory object. This
 // happens asynchronously on progress and any errors are reported
 // back on the event channel.
-func (d *Destroyer) Run(inv inventory.InventoryInfo) <-chan event.Event {
+func (d *Destroyer) Run(inv inventory.InventoryInfo, policy inventory.InventoryPolicy) <-chan event.Event {
```
[the remark this answers is not in the document]
Comment (Liujingfang1) [reply pkg/apply/destroyer.go]: Done

**Comment by Liujingfang1** [comment]
Addressed the comments. PTAL

**Review by seans3 — APPROVED** [review]
/lgtm

**Pull request merged**